packages/react/src/forms/CheckboxGroup/CheckboxGroup.stories.tsx

```diff
@@ -0,0 +1,133 @@
+import React from 'react'
+import type {Meta, StoryObj} from '@storybook/react'
+
+import {CheckboxGroup, type CheckboxGroupProps} from '.'
+import {Checkbox, FormControl, type FormValidationStatus, Stack} from '../..'
+
+type CheckboxGroupStoryProps = {
+  labelChildren: string
+  labelVisuallyHidden: boolean
+  captionChildren: string
+  validationChildren: string
+  validationVariant: FormValidationStatus
+}
+
+type MetaProps = CheckboxGroupProps & CheckboxGroupStoryProps
+
+const meta: Meta<MetaProps> = {
+  title: 'Components/Forms/CheckboxGroup',
+  component: CheckboxGroup,
+  args: {
+    labelChildren: 'Choices',
+    labelVisuallyHidden: false,
+    captionChildren: 'Select all that apply',
+    validationChildren: 'Great job!',
+    validationVariant: 'success',
+  },
+  argTypes: {
+    labelChildren: {
+      name: 'Text',
+      control: 'text',
+      table: {
+        category: 'Label',
+      },
+    },
+    labelVisuallyHidden: {
+      name: 'Visually hidden',
+      control: 'boolean',
+      table: {
+        category: 'Label',
+      },
+    },
+    captionChildren: {
+      name: 'Text',
+      control: 'text',
+      table: {
+        category: 'Caption',
+      },
+    },
+    validationChildren: {
+      name: 'Text',
+      control: 'text',
+      table: {
+        category: 'Validation',
+      },
+    },
+    validationVariant: {
+      name: 'Variant',
+      options: ['error', 'success'],
+      control: {type: 'inline-radio'},
+      table: {
+        category: 'Validation',
+      },
+    },
+  },
+}
+
+export default meta
+
+type Story = StoryObj<MetaProps>
```

Comment:
`StoryObj` is the built-in, recommended type. See the example in the [docs here](https://storybook.js.org/docs/writing-stories/typescript#typing-stories-with-meta-and-storyobj).

Reply:
Yeah, just feels redundant for us to have to declare this in every story file going forward 😕, maybe we make it a global type declaration?

Reply:
`MetaProps` is unique to every component though (not the same as Storybook's `Meta` type), so any global type that we create would have the same signature as this built-in `StoryObj` type.

```ts
// If we created our own type, it would look like this
type CustomStoryObj<T> = StoryObj<T>

// So we'd replace
type Story = StoryObj<MetaProps>

// with
type Story = CustomStoryObj<MetaProps>
```

which doesn't really have any benefit.

Reply:
> not the same as Storybook's Meta type

Ohhh now I get it.. I can't read 🤦 ... ignore me. This is totally fine 😄